cspell.json

```
@@ -324,6 +324,8 @@
         "keyvaluepairs",
         "KHTML",
         "killall",
+        "kilometres",
+        'kilometre",
```

Observed a typo; it should be double-quoted on both ends. It's single quoted on start.

```suggestion
        "kilometre",
```